src/ServiceControl.Persistence.RavenDb5/RavenDbInstaller.cs

```diff
@@ -13,8 +13,7 @@ public RavenDbInstaller(IPersistenceLifecycle lifecycle)
 
         public async Task Install(CancellationToken cancellationToken)
         {
-            await lifecycle.Start(cancellationToken);
-            await lifecycle.Stop(cancellationToken);
+            await lifecycle.Initialize(cancellationToken);
```

Comment:
Should this be disposed then?

Reply:
No, it should not. The DI container will dispose it once it is disposed.

Reply:
@DavidBoike your comment made me analyze that code again. The installer behavior is completely different. It creates its own service provider but that is not being disposed! Meaning, it is created, returns a lifecycle from that provider but that provider its lifetime isn't disposed.

Its the opposite from how regular persister is used as there a services collection is passed into the persister to register its stuff. Why isn't the same approach take for the installer? Register the installer interface in the services so that the `SetupBootstrapper` can resolve this installer via DI? This way the `CreateInstaller` isn't needed. 

We should either do our own lifetime control or not but why not do it for "runtime" but do it ourselves for "install/setup"?

Made a fix in 6c25641